x-pack/plugins/apm/server/lib/index_pattern/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Server } from 'hapi';
+// @ts-ignore
+import { getSavedObjects } from '../../../../../../src/legacy/core_plugins/kibana/server/tutorials/apm/saved_objects/get_saved_objects';
```

Comment:
Does this even need to be mentioned in the tutorial section anymore if we always auto-create these objects?

Reply:
i think it's worth showing the step in the tutorial, it's possible that the index pattern fails to save on startup, or gets deleted, so it's good visual confirmation that the step has been completed.

Reply:
Makes sense. I think it might be better to have the code live in our plugin and have the tutorial reference it rather than our plugin referencing tutorial code, although I'm not sure if there are any open source implications to that?